packages/docs/src/styles/components/syntax-highlighting.scss

```diff
@@ -1,60 +1,180 @@
 /**
- * GHColors theme by Avi Aryan (http://aviaryan.in)
- * Inspired by Github syntax coloring
+ * a11y-light theme for JavaScript, CSS, and HTML
+ * Based on the okaidia theme: https://github.com/PrismJS/prism/blob/gh-pages/themes/prism-okaidia.css
+ * @author ericwbailey
+ * @see https://github.com/ericwbailey/a11y-syntax-highlighting
  */
 
+/*
+ Theme
+ */
 code[class*='language-'],
 pre[class*='language-'] {
-  direction: ltr;
-  hyphens: none;
-  tab-size: 4;
+  color: #545454;
+  background: none;
   text-align: left;
   white-space: pre;
-  word-break: normal;
   word-spacing: normal;
+  word-break: normal;
+  word-wrap: normal;
+  line-height: 1.5;
+
+  tab-size: 4;
+  hyphens: none;
 }
 
 /* Code blocks */
 pre[class*='language-'] {
+  padding: 1em;
+  margin: 0.5em 0;
   overflow: auto;
+  border-radius: 0.3em;
+}
+
+:not(pre) > code[class*='language-'],
+pre[class*='language-'] {
+  background: #fefefe;
```

Comment:
@ericwbailey, I like that we're using your theme! I noticed on the white pages that there isn't really any contrast between the normal page background and our code blocks (`#fff` vs `#fefefe`). Do you have any design ideas for how we might show that it's a code block? Maybe an outline with a "Code" label or something? Also @line47

Reply:
I like the idea of adding a 1px border using `$color-gray-lighter` 1px solid #D6D7D9

![](https://user-images.githubusercontent.com/1449852/165296548-d79045ea-8931-491a-bbf6-1b96c65058dd.png)

Reply:
Plus one to @line47's idea!